src/main/kotlin/org/ionproject/integration/model/internal/timetable/EventCategory.kt

```diff
@@ -0,0 +1,7 @@
+package org.ionproject.integration.model.internal.timetable
+
+enum class EventCategory {
+    CLASS {
+        override fun toString() = "AULA"
```

Comment:
Still waiting for I-On Core to confirm this

Reply:
@joao-pereira5 FYI

Reply:
The identifiers for event categories are defined [here](https://github.com/i-on-project/core/blob/master/docs/api/write/insertClassSectionEvents.md#constants).

Reply:
In this case, the lecture should be marked with `2`.